atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/TimeLockMigrationEteTest.java

```diff
@@ -42,12 +51,55 @@
     private static final Gradle GRADLE_PREPARE_TASK = Gradle.ensureTaskHasRun(":atlasdb-ete-tests:prepareForEteTests");
     private static final Gradle DOCKER_TASK = Gradle.ensureTaskHasRun(":timelock-server-distribution:dockerTag");
 
+    private static final TimeLockMigrationTestCase TEST_CASE = TimeLockMigrationTestCase.CASSANDRA_PAXOS;
+    private static final TimeLockMigrationTestContext TEST_CONTEXT;
+
+    static {
+        File embeddedConfig;
+        DockerClientConfiguration dockerClientConfiguration;
+        switch (TEST_CASE) {
+            case CASSANDRA_PAXOS:
+                embeddedConfig = new File("docker/conf/atlasdb-ete.no-leader.cassandra.yml");
+                dockerClientConfiguration = ImmutableDockerClientConfiguration.builder()
+                        .initialConfigFile(embeddedConfig)
+                        .dockerComposeYmlFile(new File("docker-compose.timelock-migration.cassandra.yml"))
+                        .databaseServiceName("cassandra")
+                        .build();
+                TEST_CONTEXT = ImmutableTimeLockMigrationTestContext.builder()
+                        .eteConfigWithEmbedded(embeddedConfig)
+                        .eteConfigWithTimeLock(new File("docker/conf/atlasdb-ete.timelock.cassandra.yml"))
+                        .dockerClientConfiguration(dockerClientConfiguration)
+                        .build();
+                break;
+            case POSTGRES_DB_TIMELOCK:
```

Comment:
So this is basically a placeholder for "we will parameterize on this once we want to run the test on posgres as well"? This whole static block is a bit of a mess, you could have just put it all as separate methods of the enum though not sure if you agree that's cleaner :P

Reply:
There's an issue with shutting down the cluster on Postgres being unstable. This only applies to the shutdown part (not the assertions), and I don't have the bandwidth to fix that right now.